dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.1/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/LdapContainer.java

```diff
@@ -108,26 +107,19 @@ protected void configure() {
 
             // Add previous port bindings and KDC and LDAP ports
             Ports ports = cmd.getPortBindings();
-
-            String ldapPortMapping = String.format("%d:%d/%s", 389, 389, InternetProtocol.TCP);
-            Log.info(c, method, "adding ldap port mapping: " + ldapPortMapping);
-
-            Log.info(c, method, "PortBinding.parse(ldapPortMapping): " + PortBinding.parse(ldapPortMapping));
-            ports.add(PortBinding.parse(ldapPortMapping));
-
+            ports.bind(ExposedPort.tcp(389), Ports.Binding.empty());
```

Comment:
Suggestion: Based on your changes it looks like this entire `withCreateContainerCmdModifier` block can be replaced with
```
withExposedPort(389);
```